frontend/src/language/no.json

```
@@ -208,20 +208,20 @@
     "No predefined missions available": "Ingen tilgjengelige forhåndsdefinerte oppdrag",
     "No robot available": "Ingen robot tilgjengelig",
     "Confirm that you wish to add the selected mission(s) to the robot queue": "Bekreft at du ønsker å legge til de valgte oppdragene i robot køen",
-    "The robot is ready to run missions": "Roboten er klar til å kjøre oppdrag",
-    "The pressure measurement is not currently available, so start the mission at your own risk.": "Trykkmålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
+    "Robot ready to run missions": "Roboten er klar til å kjøre oppdrag",
+    "Pressure measurement currently unavailable, proceed with caution.": "Trykkmålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
```

```suggestion
    "Pressure measurement currently unavailable, proceed with caution.": "Trykkmåling ikke tilgjengelig, start oppdrag på egen risiko.",
```
Just to make it more similar to the English